packages/discovery/src/discovery/provider/multicall/MulticallClient.ts

```diff
@@ -17,7 +17,7 @@ export interface CallProvider {
 export class MulticallClient {
   constructor(
     private readonly provider: CallProvider,
-    private readonly config: MulticallConfig,
+    private readonly config?: MulticallConfig,
```

Comment:
Since we're passing this config as a function argument later it's better to remove this variable as a field of this class so we can't access it through `this.config` as a mistake